atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -117,7 +117,8 @@ protected Transaction startTransaction() {
                 AbstractTransactionTest.GET_RANGES_EXECUTOR,
                 AbstractTransactionTest.DEFAULT_GET_RANGES_CONCURRENCY,
                 getSweepQueueWriterInitialized(),
-                MoreExecutors.newDirectExecutorService()) {
+                MoreExecutors.newDirectExecutorService(),
+                () -> false) {
```

Comment:
When we're passing `() -> false` in, we could pass in something like `CommitLogProcessor.neverProfile()`